cmd/fuzzbinder/README.md

```diff
@@ -1 +1,64 @@
 # Fuzzing Claims for revisions of a source code
+
+The goal of Fuzzbinder is to generated, sign and publish in a transparency log fuzzing claims for revisions of the source code of a given project to endorse their security.
+
+By following the steps described below, you will be able to generate fuzzing claims for your project using the fuzzing reports generated by OSS-Fuzz and ClusterFuzz for your project, and the command line tool
+[`cmd/fuzzbinder`](/cmd/fuzzbinder/) that we have provided to generate fuzzing claims.
+
+## Generating fuzzing claims using the `cmd/fuzzbinder` tool
+
+If you have not already done so, consult the [`development guidelines`](./../../docs/development-guidelines.md) to find out what you need to
+install before following the steps below.
+
+### 1) Establish access to GCS
+
+To get the fuzzing statistics, you should have access to the fuzzing metadata generated by OSS-Fuzz and ClusterFuzz for your project. Set up `gsutil` and ensure that you have access to the fuzzing metadata by doing the following:
+
+- Install the [gsutil tool](https://cloud.google.com/storage/docs/gsutil_install).
+- Check whether you have access to the fuzzing logs and coverage reports of your project:
+
+```sh
+$ gsutil ls gs://oss-fuzz-coverage/${PROJECT_NAME}
+```
+
+```sh
+$ gsutil ls gs://${PROJECT_NAME}-logs.clusterfuzz-external.appspot.com
+```
+
+If you get an authorization error from the commands above, run the following commands:
+
+```sh
+$ gcloud auth application-default login
+```
+
+```sh
+$ gcloud auth login
+```
+
+If you still have authentication problems look at this [authentication guide](https://googleapis.dev/python/google-api-core/latest/auth.html) and try again. Once `gsutil` works, you can run the fuzzing claim generation command.
+
+### 2) Generate fuzzing claim
+
+The [`fuzzbinder`](/internal/fuzzbinder/) package provides functionality for generating a fuzzing claim file.
+
+To generate a fuzzing claim run:
+
+```sh
+$ go run cmd/fuzzbinder/main.go -project_name <project-name> \
+   -homepage <path-to-project-git-repo-root>  \
+  -fuzzengine <fuzzing-engine> -sanitizer <fuzzing-sanitizer>  \
+  -date <fuzzing-date> -fuzzclaim_path <fuzzclaim-path>
+```
+
+As you may noticed, you will need `<fuzzing-engine>` and `<fuzzing-sanitizer>` that you used in your OSS-Fuzz project configuration.
+For instance, if you used `libfuzzer` as a fuzzing engine and an `address sanitizer` for your project, and you want to generate a fuzzing claim
```

Comment:
Some of this info should be included in the help string of the command flags. Please see my comments on that.